### PR TITLE
fix(challenges): D3 challenges

### PR DESCRIPTION
**Pull request by scissorsneedfoodtoo**
Added an updated version of D3 as a CDN link to the data-visualization-with-d3 meta file and did the same for the latest version of Font Awesome in the Bootstrap challenges meta file. Doing this fixes the D3 challenges along with a couple of challenges in the Bootstrap section that require FA.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by ValeraS** [comment]
All D3 challenges contain a required link. If we add a link into the `meta` file we should remove links from challenge files.

**Comment by scissorsneedfoodtoo** [comment]
@ValeraS, yes, I agree. I started doing that for all the D3 files in all languages, but found that it broke the test suite. Started playing with the test-challenges.js file to try and point to the required link in the meta but figured it was a bit outside the scope of this PR. I'd be happy to go back and make those changes, though

**Comment by ValeraS** [comment]
Rebase to latest master and try remove links again. It should work now.

**Comment by scissorsneedfoodtoo** [comment]
Sure! I'll do that and update this PR as soon as I can.

**Comment by scissorsneedfoodtoo** [comment]
@ValeraS, would you mind taking another look at this when you get a chance? And please let me know if there's anything else I can change.

File: curriculum/challenges/_meta/bootstrap/meta.json [hunk 1]
```diff
@@ -7,6 +7,9 @@
   "required": [
     {
       "link": "https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.css"
+    },
+    {
+      "link": "https://use.fontawesome.com/releases/v5.5.0/css/all.css"
```
Comment (ValeraS) [inline curriculum/challenges/_meta/bootstrap/meta.json]: IMO, it's not needed here. Only a few challenges require FA and they contain a link to it. Now both links from a `meta` file and from a challenge file are used in the client and for test challenges. Let's remove it from this PR and if you want you'll open new PR with updated links in challenge files.

**Review by ValeraS — CHANGES_REQUESTED** [review]

**Comment by scissorsneedfoodtoo** [comment]
@ValeraS, yes, I agree. Glad to see that the required field is being loaded to the iframe now! I learned a lot seeing how you did that.

I went ahead and made the requested change and this should be ready for further review.

**Comment by RandellDawson** [comment]
@ValeraS What do you think about these changes?

**Review by ValeraS — APPROVED** [review]

**Pull request merged**